plugin/arcconf/arcconf.py

```diff
@@ -14,7 +14,6 @@
 # License along with this library; If not, see <http://www.gnu.org/licenses/>.
 #
 # Author: Gris Ge <fge@redhat.com>
-#         Joe Handzik <joseph.t.handzik@hpe.com>
```

Comment:
Ok. Got it. Initially, I had done the same. 
I had appended my name under the author and had HPE in the copyright as is. I mistook the first comment about the HPE copyright. Sorry for that.